src/components/partial-string-match/PartialStringMatch.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+import {PartialStringMatch} from './PartialStringMatch';
+
+describe('PartialStringMatch', () => {
+    const testString = 'test-string';
+
+    it('should render an empty span if the wholeString is a falsy value', () => {
+        [undefined, null, ''].forEach((falsyVal) => {
+            expect(shallow(<PartialStringMatch wholeString={falsyVal} />).find('span').text())

```

Comment:
Can you use `text()` on the component instead of finding the span first?

AKA `.text()` instead of `.find('span').text()`

Reply:
I did it the way you mention at some point, but I reverted telling myself I wanted to test everything is wrapped in a span as well